Test.java

```diff
@@ -0,0 +1,12 @@
+class MyClass {
+  MyClass(MyClass mc) { }
+  
+  int     foo1() { return 0; }
+  void    foo2(int value) { }
+  int     foo3(int value) { return 0; } // Noncompliant
+  Object  foo4(int value) { return null; }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "value". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1172)